components/director/internal/domain/formation/service.go

```diff
@@ -1509,7 +1513,7 @@ func (s *service) DeleteAutomaticScenarioAssignment(ctx context.Context, in *mod
 // RemoveAssignedScenarios removes all the scenarios that are coming from any of the provided ASAs
 func (s *service) RemoveAssignedScenarios(ctx context.Context, in []*model.AutomaticScenarioAssignment) error {
```

Comment:
Can we rename the method to something like `UnassignFormationsCommingFromASA`. The current name is a bit missleading.

Reply:
Same applies to the asaEngine method as well